Task5.c

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+char* getString()
+{
+    int strLength = 0;
+    int strCapacity = 1; // place for '\0' 
+    char* string = malloc(sizeof(char));
+    char scannedChar = getchar();
+    while (scannedChar != '\n')
+    {
+        string[strLength] = scannedChar;
+        strLength++;
+        if (strLength >= strCapacity)
+        {
+            strCapacity *= 2;
+            string = realloc(string, strCapacity * sizeof(char));
+        }
+        scannedChar = getchar();
+    }
+    string[strLength] = '\0';
+    return string;
+}
+
+int bracketsType(char bracket1, int bracket2)
```

Comment:
```suggestion
bool bracketsType(char bracket1, int bracket2)
```
Возвращаем да или нет, а значит тип bool